sunxspex/thermal.py

```diff
@@ -0,0 +1,699 @@
+import copy
+
+import astropy.units as u
+import numpy as np
+from scipy import interpolate, stats
+from sunpy.data import manager
+
+from sunxspex.io import load_chianti_continuum, load_chianti_lines_lite, load_xray_abundances
+
+doc_string_params = """
+Parameters
+----------
+energy_edges: `astropy.units.Quantity`
+    The edges of the energy bins in a 1D N+1 quantity.
+
+temperature: `astropy.units.Quantity`
+    The temperature of the plasma.
+    Can be scalar or 1D of any length. If not scalar, the flux for each temperature
+    will be calculated. The first dimension of the output flux will correspond
+    to temperature.
+
+emission_measure: `astropy.units.Quantity`
+    The emission measure of the plasma at each temperature.
+    Must be same length as temperature or scalar.
+
+abundance_type: `str` (optional)
```

Comment:
I think that we should explicitly state here that these abundance_type strings are legacy from the IDL version, as people may wonder what mewe etc is.

Reply:
Is there a document somewhere that describes what each abundance set is?  If so, we should probably just point to that or copy text from it.